dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05390MakeRoomForLongerJobDetailTest.java

```diff
@@ -0,0 +1,39 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.db.DbType;
+import com.dotmarketing.util.Logger;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class Task05390MakeRoomForLongerJobDetailTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6309437)